pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -235,22 +235,22 @@ func checkVolumeSatisfyClaim(volume *v1.PersistentVolume, claim *v1.PersistentVo
 	requestedSize := requestedQty.Value()
 	isMisMatch, err := checkVolumeModeMisMatches(&claim.Spec, &volume.Spec)
 	if err != nil {
-		return fmt.Errorf("error checking if volumeMode was a mismatch: %v", err)
+		return fmt.Errorf("error checking volumeMode: %v", err)
 	}
 
 	volumeQty := volume.Spec.Capacity[v1.ResourceStorage]
 	volumeSize := volumeQty.Value()
 	if volumeSize < requestedSize {
-		return fmt.Errorf("Storage capacity of volume[%s] requested by claim[%v] is not enough", volume.Name, claimToClaimKey(claim))
+		return fmt.Errorf("requested PV is too small")
 	}
 
 	requestedClass := v1helper.GetPersistentVolumeClaimClass(claim)
 	if v1helper.GetPersistentVolumeClass(volume) != requestedClass {
-		return fmt.Errorf("Class of volume[%s] is not the same as claim[%v]", volume.Name, claimToClaimKey(claim))
+		return fmt.Errorf("storageClasseNames do not match")
```

Comment:
Why does the storage class name take plural?
Should we return `storageClassName does not match`?

Reply:
maybe just like， your password and confirmation password do not match.

Reply:
Fixed to singular.

Reply:
>> Fixed to singular.

Then, the message is storageClassName `does` not match, right?